domains/anthony.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Anthony's personal developer website",
+  "repo": "https://github.com/github-username/github-username.github.io",
```

Comment:
If you are using github pages to host the website, please correct the link. Otherwise, you can remove the `repo` from the json file.

Reply:
Removed that, thanks.